src/transformers/__init__.py

```diff
@@ -2314,6 +2315,7 @@
             "TFLayoutLMForMaskedLM",
             "TFLayoutLMForSequenceClassification",
             "TFLayoutLMForTokenClassification",
+            # XXX "TFLayoutLMForQuestionAnswering",
```

Comment:
To be done in a separate PR?

Reply:
Yes, let's do it in a separate PR. I'll remove these commented out values.